deployments/deployments.go

```diff
@@ -0,0 +1,94 @@
+// Package deployments provides functions for decoding V2 paasta deployments
+// of the form:
+//   "v2": {
+//    "deployments": {
+//		"everything": {
+//		  "docker_image": "services-fluffy:paasta-abcdfff",
+//		  "git_sha": "abcdfff"
+//		},
+//		"stagef": {
+//		  "docker_image": "services-fluffy:paasta-abcdfff",
+//		  "git_sha": "abcdfff"
+//		}
+//	  },
+//  }
+package deployments
+
+import (
+	"fmt"
+
+	paastaconfig "github.com/Yelp/paasta-tools-go/config"
+)
+
+// V2DeploymentGroup ...
+type V2DeploymentGroup struct {
+	DockerImage string `json:"docker_image"`
+	GitSHA      string `json:"git_sha"`
+}
+
+// V2DeploymentsConfig ...
+type V2DeploymentsConfig struct {
+	Deployments map[string]V2DeploymentGroup `json:"deployments"`
+}
+
+// Deployments ...
+type Deployments struct {
+	V2 V2DeploymentsConfig `json:"v2"`
+}
+
+// DockerRegistry ...
+type DockerRegistry struct {
+	Registry string `json:"docker_registry"`
+}
+
+func getDockerRegistry() (string, error) {
+	dockerRegistry := &DockerRegistry{}
+	configreader := paastaconfig.SystemPaaSTAConfigFileReader{
```

Comment:
this doesn't look right to me - won't this end up trying to read `/etc/paasta/docker_registry.json/deployments.json`?

Reply:
My bad, seems like an oversight. Fixed. (I will also test this locally once)